src/main/java/com/intuit/graphql/orchestrator/utils/GraphQLUtil.java

```diff
@@ -82,4 +90,33 @@ public static Type createTypeBasedOnGraphQLType(GraphQLType graphQLType) {
     throw new CreateTypeException(
         String.format(ERR_CREATE_TYPE_UNEXPECTED_TYPE, GraphQLTypeUtil.simplePrint(graphQLType)));
   }
+
+  public static String getUniqueIdFromFieldSet(String fieldSet) {
```

Comment:
I think we should move this to FederationUtils.javaa

Reply:
Should be in the next external/refactor pr